packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -142,10 +169,10 @@ export function trackViews(location: Location, lifeCycle: LifeCycle, domMutation
       currentView = trackViewChange(startClocks, name)
     },
     stop: () => {
+      stopAutomaticViewCollection && stopAutomaticViewCollection()
```

Comment:
Let the bundler do the minification for us:
```suggestion
      if (stopAutomaticViewCollection) stopAutomaticViewCollection()
```
(you could also initialize it to `noop` to simplify)

Reply:
the issue disappear with latest change 😉